tests/openebs-pool-container-failure_test.go

```diff
@@ -110,112 +114,85 @@ var _ = Describe("BDD of openebs pool container failure experiment", func() {
 				}
 			}
 
-			fmt.Printf("ContainerIDs before chaos has been recorded\n")
-			fmt.Printf("Container Restart count before chaos has been recorded\n")
+			fmt.Println("ContainerIDs before chaos has been recorded")
+			fmt.Println("Container Restart count before chaos has been recorded")
 
-			//Fetching rbac file
-			By("Fetching rbac file for the experiment")
+			//Installing RBAC for the experiment
+			//Fetching RBAC file
+			By("Installing RBAC for the experiment")
 			err = exec.Command("wget", "-O", "pool-container-failure-sa.yaml", "https://raw.githubusercontent.com/litmuschaos/chaos-charts/master/charts/openebs/openebs-pool-container-failure/rbac.yaml").Run()
-			Expect(err).To(BeNil(), "failed to create rbac")
-			if err != nil {
-				fmt.Println(err)
-			}
-
-			//Modify Namespace field of the rbac
-			By("Modify Namespace field of the rbac")
+			Expect(err).To(BeNil(), "failed to fetch RBAC file")
+			//Modify Namespace field of the RBAC
 			err = exec.Command("sed", "-i", `s/namespace: default/namespace: litmus/g`, "pool-container-failure-sa.yaml").Run()
-			Expect(err).To(BeNil(), "failed to create rbac")
-			if err != nil {
-				fmt.Println(err)
-			}
-
-			//Creating rbac for the experiment
-			By("Creating rbac for the experiment")
+			Expect(err).To(BeNil(), "fail to modify namespace field of the RBAC")
 			err = exec.Command("kubectl", "apply", "-f", "pool-container-failure-sa.yaml").Run()
-			Expect(err).To(BeNil(), "fail to create chaos experiment")
+			Expect(err).To(BeNil(), "fail to create RBAC for the experiment")
 			if err != nil {
 				fmt.Println(err)
 			}
 
 			//Creating Chaos-Experiment
 			By("Creating Experiment")
-			err = exec.Command("kubectl", "apply", "-f", "https://hub.litmuschaos.io/api/chaos?file=charts/openebs/openebs-pool-container-failure/experiment.yaml", "-n", "litmus").Run()
+			err = exec.Command("kubectl", "apply", "-f", "https://hub.litmuschaos.io/api/chaos?file=charts/openebs/openebs-pool-container-failure/experiment.yaml", "-n", chaosNamespace).Run()
 			Expect(err).To(BeNil(), "fail to create chaos experiment")
 			if err != nil {
 				fmt.Println(err)
 			}
 
 			fmt.Println("Chaos Experiment Created Successfully")
 
-			//Creating chaosEngine
+			//Installing chaos engine for the experiment
+			//Fetching engine file
+			By("Fetching engine file for the experiment")
+			err = exec.Command("wget", "-O", "pool-container-failure-ce.yaml", "https://raw.githubusercontent.com/litmuschaos/chaos-charts/master/charts/openebs/openebs-pool-container-failure/engine.yaml").Run()
+			Expect(err).To(BeNil(), "Fail to fetch engine file")
+			//Modify chaos engine spec
+			//Modify Namespace of the engine
+			err = exec.Command("sed", "-i", `s/namespace: default/namespace: litmus/g`, "pool-container-failure-ce.yaml").Run()
```

Comment:
Why are you using separate sed commands for every variable? Can you please do the same thing in only one command? Do the same thing for all files?

Reply:
we can but then that will be a long command and bit complex to understand check even though it is ok.

Reply:
done